Exercise_B/index.css

```diff
@@ -2,3 +2,36 @@
 This css sheet is linked to index.html
 You can style tags, classes (.) or ids (#) from this file.
 */
+
+div {
+    width: 100px;
+    height: 100px;
+}
+
+.box {
+    position: relative;
```

Comment:
you can add 
border: solid 5px black;
inside the .box so that you won't need to write it again for each box